n2kMapper.js

```diff
@@ -76,6 +80,12 @@ var toDelta = function (n2k, state) {
         }
       ]
     }
+    if ( src_state && src_state.canName ) {
+      result.updates[0].source.canName = src_state.canName
+    }
+    if ( src_state && typeof src_state.deviceInstance !== 'undefined' ) {
+      result.updates[0].source.deviceInstance = src_state.deviceInstance
```

Comment:
What's `deviceInstance` - as opposed to `instance` that we are already using?

Reply:
This is different. Every device on the system will have a deviceInstance. This is not the same as the "instance" on PGNs for things like tanks and batteries. These should be unique for all devices of the same class and function.